setup.cfg

```diff
@@ -14,7 +14,7 @@ markers =
 # E203 false positive, see https://github.com/PyCQA/pycodestyle/issues/373
 # W605 false positive until python3.8: https://github.com/PyCQA/pycodestyle/issues/755
 ignore = F999,E203,E501,E128,E124,E402,W503,W504,W605,E731,C901,B007,B306,B009,B010
-exclude = .venv/.git,*/migrations/*,node_modules/*,src/sentry/static/sentry/vendor/*,docs/*,src/south/*,examples/*
```

Comment:
seems unrelated?

Reply:
Yeah, this was jsut something I saw and wanted to correct. `.venv/.git` is concerning but then again no one runs `flake8` direct, so it's not really a big issue.